go-controller/pkg/node/gateway_udn.go

```diff
@@ -437,11 +437,17 @@ func (udng *UserDefinedNetworkGateway) DelNetwork() error {
 			return fmt.Errorf("failed to reconcile default gateway for network %s, err: %v", udng.GetNetworkName(), err)
 		}
 	}
+
+	err := udng.updateAdvertisedUDNIsolationRules(false)
+	if err != nil {
+		return err
```

Comment:
can we have a better custom error to track what went wrong
fmt.Errorf("failed to delete rules to drop traffic from host to advertised UDNs for network blah" something
why do we rely on having this in del but not in add? I'd think doreconcile should take care of del as well no?